src/routes/(app)/order/[id]/fetchOrderForUser.ts

```diff
@@ -137,7 +137,8 @@ export async function fetchOrderForUser(orderId: string) {
 				shipping: item.product.shipping,
 				paymentMethods: item.product.paymentMethods,
 				isTicket: item.product.isTicket,
-				variationLabels: item.product.variationLabels
+				variationLabels: item.product.variationLabels,
+				externalResource: item.product.externalResource
```

Comment:
This should only be fetched if the order is paid.

You can remove the field after fetching the order if the order is not fully paid.

Otherwise someone can monitor the network call and access the resource without paying